policyengine_us/variables/gov/states/az/tax/income/exemptions/az_parents_grandparents_exmeption.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class az_parents_grandparents_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
```

Comment:
would suggest making this person-level since it's just summing up characteristics of person-level. the `adds` in the exemption variable will sum the person-level amounts

Reply:
don't sum eligible people in the formula, that'll result in double-counting when aggregating the person-level variable to tax unit